analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -545,11 +546,13 @@ def determine_compiler(gcc_command):
     gcc_command -- A split build action as a list which may or may not start
                    with ccache.
 
+    In the third case gcc is a symlink to ccache, but we can handle it as a normal
+    compiler.
```

Comment:
Please, move these two lines above `gcc_command -- ...` because that is the function parameter description which should be at the end of the function's description.